stan/inc2prev_antibodies.stan

```diff
@@ -70,7 +72,7 @@ parameters {
 }
 
 transformed parameters {
-  vector[t] gp; // value of gp at time t
+  vector[diff_order == 0 ? t : t-1] gp; // value of gp at time t
```

Comment:
Should this be `t - diff_order`? Seems to me that if the GP prior applies to the differences in `r` then it would need an initial value of `r` otherwise I think it's assumed to be 0.

Reply:
This is true, I think as written the first n terms of the GP will just have a slightly different definition to the rest (so when diff_order = 2, the first step will be the difference from 0 to the first r estimate and from then it will be r to r difference). The benefit of leaving as is is really just code simplicity and TBH I think it may not make much difference...